_posts/2019-07-11-deploy-spring-boot-kubernetes.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+authors: [yolan_vloeberghs]
+title: 'Deploying your Spring Boot application in the cloud with Kubernetes'
+image: /img/2019-07-11-deploy-spring-boot-kubernetes/banner.jpg
+tags: [Kubernetes,Spring,Spring Boot,Docker,AWS,EKS,Google,GKE,DigitalOcean,Azure,AKS,kubectl,Minikube]
+category: Cloud
+comments: true
+---
+
+# Deploying your Spring Boot application in the cloud with Kubernetes
+
+Assuming you have already heard of Kubernetes, you are probably aware of the continuing growth of this platform. More and more platforms are growing in popularity because of the flexibility of these technologies. Look at OpenShift, Cloud Foundry, ....
+
+As adaptation is growing, many developers are wondering how they effectively use this platform to deploy their application in the cloud with a Kubernetes cluster and use its benefits.
```

Comment:
As adaptation is growing, many developers are wondering how to effectively use these platforms to deploy their application in the cloud on a kubernetes cluster and make full use of its benefits.